### PR TITLE
Identity pallet

### PR DESCRIPTION
**Pull request by kostekIV**
# Description
Adds an identity pallet

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

# Checklist:

- I have bumped `spec_version` and `transaction_version`

### CODE REVIEW

File: bin/runtime/src/lib.rs [hunk 1]
```diff
@@ -670,6 +670,30 @@ impl pallet_contracts::Config for Runtime {
     type RelaxedMaxCodeLen = ConstU32<{ 256 * 1024 }>;
 }
 
+parameter_types! {
+    pub const BasicDeposit: Balance = 258 * DEPOSIT_PER_BYTE;
+    pub const FieldDeposit: Balance = 66 * DEPOSIT_PER_BYTE;
+    pub const SubAccountDeposit: Balance = 85 * DEPOSIT_PER_BYTE;
```
Comment (pmikolajczyk41) [inline bin/runtime/src/lib.rs:676]: it would be good to provide some explanation/calculations behind these values

Comment (pmikolajczyk41) [reply bin/runtime/src/lib.rs:676]: for now it should be sufficient

Comment (kostekIV) [reply bin/runtime/src/lib.rs]: done

File: pallets/elections/src/impls.rs [hunk 2]
```diff
@@ -414,7 +414,7 @@ mod tests {
                     reserved.clone(),
                     non_reserved.clone(),
                 )
-                .expect("Expected non-empty rotated committee!")
+                .lletexpect("Expected non-empty rotated committee!")
```
Comment (pmikolajczyk41) [inline pallets/elections/src/impls.rs:417]: O_O

**Review by pmikolajczyk41 — APPROVED** [review]
there is nothing to complain about it, so just two points:
 - what is the reason for adding it now? some Jira?
 - pallet's README is not very explanatory, so it would be good to create some documentation for us about it; it doesn't have to be long but at least some terminology (registrars etc) and some examples would be great (in particular, I do not see any UI support for this, so it's not so intuitive how to use it)

**Review by mike1729 — APPROVED** [review]

File: bin/runtime/src/lib.rs [hunk 3]
```diff
@@ -112,7 +112,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 17,
+    spec_version: 18,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 6,
```
Comment (DamianStraszak) [inline bin/runtime/src/lib.rs:118]: Checklist says you bumped `transaction_version` :)

**Review by DamianStraszak — CHANGES_REQUESTED** [review]

**Review by DamianStraszak — APPROVED** [review]

**Pull request merged**